packages/components/src/theme/types.ts

```diff
@@ -0,0 +1,5 @@
+export const SIZES = ['xs', 'sm', 'md', 'lg'] as const;
```

Comment:
what's this file?

Reply:
It is the same that I defined on Button component and I use it to have some constants as different sizes exist